src/server/routes/index.js

```diff
@@ -52,15 +52,25 @@ router.get('/', async (req, res, next) => {
 			requireJS: Boolean(res.locals.user)
 		});
 
+		const disableSignUp = props.alerts.some(
```

Comment:
A safer way to do this is set `req.signUpDisabled` (renamed for clarity) in `app.js` in this block: 
https://github.com/bookbrainz/bookbrainz-site/blob/master/src/server/app.js#L132-L138

Then instead of doing `props.alerts.some(…`, you can check `req.signUpDisabled`